elide-core/src/main/java/com/yahoo/elide/audit/LogMessage.java

```diff
@@ -120,6 +121,19 @@ public String getMessage() {
                 ctx.setVariable(name, expression);
                 singleElementContext.setVariable(name, singleElementExpression);
             }
+
+            final User user = record.getRequestScope().getUser();
```

Comment:
Changes look good.  Can we move this snippet to a helper function?  This function is already pretty large with high cyclomatic complexity.

Reply:
Done.